docs/release-notes/release-notes-0.16.1.md

```diff
@@ -69,6 +69,14 @@ https://github.com/lightningnetwork/lnd/pull/7359)
   anything for the users since the message type(36) stays unchanged, except in
   the logging all the appearance of `funding_locked` replated experssion is
   replaced with `channel_ready`.
+
+* [Fix non-deterministic behaviour in RPC calls for
```

Comment:
Sorry, it looks like we missed 0.16.1 as the first RC is now out. Can you move this to the 0.17 file please?

Reply:
Sure no worries! 🙂